test/cookie-specs.js

```diff
@@ -11,66 +11,62 @@ chai.use(chaiAsPromised);
 chai.expect();
 let expect = chai.expect;
 
-describe ('cookies.js', () => { 
-  it ('should properly read a cookie and return the value given the key', () => {
-    let cookieMan = new Cookie('k=v');
-    cookieMan.readCookie('k').should.equal('v');
+describe ('cookies.js', () => {
+  it ('should properly create a JS cookie', () => {
```

Comment:
all of these `it` blocks seem to have a space in-between `it` and the call parentheses. let's get rid of that space in all these cases.